code/modules/mafia/roles/roles.dm

```diff
@@ -49,21 +52,29 @@
 
 /datum/mafia_role/New(datum/mafia_controller/game)
 	. = ..()
+	mafia_panel = new(null, game)
 	for(var/datum/mafia_ability/abilities as anything in role_unique_actions + /datum/mafia_ability/voting)
 		role_unique_actions += new abilities(game, src)
 		role_unique_actions -= abilities
 
 /datum/mafia_role/Destroy(force, ...)
 	UnregisterSignal(body, COMSIG_MOB_SAY)
```

Comment:
This will runtime if the body was somehow precociously deleted, and register_body doesn't seem to take that into account.

Reply:
It shouldn't be doing so, so if it is I'd prefer it to runtime so I can look into why that's happened